packages/devextreme/js/events/pointer.js

```diff
@@ -93,6 +99,17 @@ const pointer = {
     out: 'dxpointerout'
 };
 
+function getStrategyFromGlobalConfig(name) {
+    const config = GlobalConfig();
```

Comment:
maybe make one line? const config = GlobalConfig().pointerEventStrategy;

Reply:
yep..it's better